src/main/java/org/minions/devfund/damianvp/battleship/Ocean.java

```diff
@@ -0,0 +1,189 @@
+package org.minions.devfund.damianvp.battleship;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+/**
+ * Ocean class.
+ */
+public class Ocean {
+    static final int LENGTH = 20;
+    static final int TOTAL_SHIPS = 13;
+    private Ship[][] ships = new Ship[LENGTH][LENGTH];
+    private int shotsFired;
+    private int hitCount;
+    private int shipsSunk;
+    private Map<String, Integer> fleet;
+
+    /**
+     * Ocean constructor.
+     */
+    Ocean() {
+        for (int i = 0; i < LENGTH; i++) {
+            for (int j = 0; j < LENGTH; j++) {
+                EmptySea emptySea = new EmptySea();
+                this.ships[i][j] = emptySea;
+                emptySea.placeShipAt(i, j, true, this);
+            }
+        }
+        this.shotsFired = 0;
+        this.hitCount = 0;
+        this.shipsSunk = 0;
```

Comment:
Integer attributes of class are 0 by default.
They are redundant

Reply:
ok, I will remove those